Assets/_BForBoss/_Utility/Scripts/PGInputSystem.cs

```diff
@@ -0,0 +1,147 @@
+using System;
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+namespace Perigon.Utility
```

Comment:
Wraps all the InputSystem in one object/place

Hides all the complex Unity Input System, and keeps things simple for other components to subscribe to this.